app/templates/index.html

```diff
@@ -100,5 +100,18 @@
 {% include "toolkit/browse-list.html" %}
 {% endwith %}
 
+{% if current_user.has_any_role('admin-ccs-sourcing') %}
```

Comment:
Is it worth having the dashboard link for regular admins too?  Probably not? I don't think Nicole should be getting involved with these at all.